packages/client/hmi-client/src/router/index.ts

```diff
@@ -16,8 +17,8 @@ export enum RoutePath {
 	DocView = '/docs/:id?',
 	Project = '/projects/:projectId',
 	ModelView = '/projects/:projectId/model/:modelId',
-	SimulationView = '/projects/:projectId/simulation',
-	Results = '/projects/:projectId/results',
+	SimulationView = '/projects/:projectId/simulations/:simulationId?',
+	SimulationResultView = '/projects/:projectId/simulation-results/:simulationRunId?',
```

Comment:
Why the change to _Simulation Result_ instead of _Result_?
Are we planning on showing different kind of results in the future?

Reply:
Just making it more explicit, because we will have `datasets` that can be simulation-results.

Reply:
But if datasets can be simulation results, we can display them on the same view? We will have a control system to switch between tabular and matrix cells views.